packages/editor-ui/package.json

```diff
@@ -46,6 +46,7 @@
     "@jsplumb/core": "^5.13.2",
     "@jsplumb/util": "^5.13.2",
     "@lezer/common": "^1.0.4",
+    "@n8n/chat": "^0.1.4",
```

Comment:
This was intentionally left out to avoid adding this package as a dependency in any of the releases so far. The error in the console is actually a warning that shouldn't be breaking anything.

Reply:
> to avoid adding this package as a dependency in any of the releases so far

Why don't we want to add it?

Reply:
I think we want to add it once the AI feature is actually merged.